coops-ui/src/components/AnnualReport/Certify.vue

```diff
@@ -22,23 +22,21 @@
         <template slot="label">
           <div class="certify-stmt">
             I, <b>{{trimmedCertifiedBy || '[Legal Name]'}}</b>, certify that I have relevant knowledge of the
-            association and that I am authorized to make this filing.
+            {{ entityDisplay || 'association' }} and that I am authorized to make this filing.
           </div>
         </template>
       </v-checkbox>
       <p class="certify-clause">{{currentDate}}</p>
       <p class="certify-clause">
-        Note: It is an offence to make a false or misleading statement in
-        respect of a material fact in a record submitted to the Corporate Registry for filing.
-        See section 200 of the Cooperatives Association Act.
+        {{message}}
       </p>
     </div>
   </v-card>
 </template>
 
 <script lang="ts">
 import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
-
+import { configJson } from '@/resources'
```

Comment:
Maybe just add a short comment above the imports like // Components  ...  // Enums   etc etc. Just to give a quick idea of what exactly the configJson is.